UnityProject/Assets/Scripts/Objects/Closets/RechargeDock.cs

```diff
@@ -0,0 +1,96 @@
+using System.Collections;
+using System.Collections.Generic;
+using HealthV2;
+using Objects;
+using Systems.Electricity;
+using UnityEngine;
+
+public class RechargeDock : ClosetControl, IAPCPowerable
+{
+	public float IdleWattage = 1f;
+
+	public float ChargingWattage = 300f;
+
+
+	public IChargeable ChargeableDevice;
+
+	public APCPoweredDevice APCPoweredDevice;
```

Comment:
looks like you are setting this field in  awake, why make it ``public`` then? It only pollutes the component view in inspector with fields you aren't meant to fill and nothing tells you that.